packages/forklift-console-plugin/webpack.config.ts

```diff
@@ -88,12 +88,23 @@ const config: WebpackConfiguration & {
       patterns: [{ from: '../locales', to: '../dist/locales' }],
     }),
     new EnvironmentPlugin({
+      // DATA_SOURCE: used for testing when no api servers are available
+      // if set to mock, network api calls will be mocked
       DATA_SOURCE: 'remote',
+      // can be 'RedHat' or 'Konveyor',
+      // note: downstream build are set to: 'RedHat'
       BRAND_TYPE: 'Konveyor',
+      // NAMESPACE: used only on mock data
       NAMESPACE: 'konveyor-forklift',
-      DEFAULT_NAMESPACE: 'default',
+      // DEFAULT_NAMESPACE: UI forms and modals will fallback to this namespace
+      //                    if no namespace is given by user.
+      // note: downstream build are set to: 'openshift-mtv'
+      DEFAULT_NAMESPACE: 'konveyor-forklift',
```

Comment:
@ahadas @mnecas hi,

checking for the installation namespace is hard, so instead of that i'll 
a. use the `DEFAULT_NAMESPACE` set at compile time by @mnecas to be `openshift-mtv`, Martin please keep me honest here.
b. let the user use a different namespace when using the UI if they need to.